lighthouse-core/gather/driver.js

```diff
@@ -752,6 +795,7 @@ class Driver {
     // Wait for both load promises. Resolves on cleanup function the clears load
     // timeout timer.
     const loadPromise = Promise.all([
+      waitForSecurityCheck.promise,
```

Comment:
I think it'd only work if the `Security` domain is disabled before `waitForPageLoad` runs. If a gatherer were to enable `Security` and forget to disable it, this may break?

Also, isn't this method always called from "about:" -> "target url"?

Reply:
> Also, isn't this method always called from "about:" -> "target url"?

That's kinda what I'm talking about, just explaining why this is always going to be OK :)

Reply:
Ah, OK I misunderstood. How's the comment I added?